recipes/cad1/task1/baseline/enhance.py

```diff
@@ -0,0 +1,309 @@
+""" Run the dummy enhancement. """
+import json
+import logging
+from pathlib import Path
+from typing import Dict, List, Tuple
```

Comment:
Will need to import `Union` too for function definition typing.

```suggestion
from typing import Dict, List, Tuple, Union
```